src/Leaflet.VectorGrid.js

```diff
@@ -5,6 +5,7 @@ L.VectorGrid = L.GridLayer.extend({
 	options: {
 		rendererFactory: L.svg.tile,
 		vectorTileLayerStyles: {},
+		pointFeatureHooks: [],
```

Comment:
A few reasons why I did it this way:
  - the concept of feature in a vt seems less clear than in geojson to me
  - vt have layers not geojson, it separates things neatly
  - I see it more like vectorTileLayerStyles functions, it is external styling by the user
  - It may not imitate L.GeoJSON but it imitates VectorGrid existing options
  - a single callback would be executed for very dense vt layers that the user may not be interested in. This includes the user's code and the pre-processing on our side (like transforming tile coords into map coords)